admin/publish.md

```diff
@@ -38,6 +38,8 @@ If there are no errors, you can start preparing for the new release.
 
 Generate a GitHub auth token by going to https://github.com/settings/tokens. Save the token somewhere for future reference.
 
+> Before running the command below, get all tags from the original repository by runnung `git fetch --tags`.
```

Comment:
Hi @lex111 

Do you remember why this is needed? 

I removed it, because I asked @yangshun and we think Github labels and git tags are not related, so this is likely not needed.